Permanent/Network/APIErrors.swift

```diff
@@ -56,3 +56,23 @@ enum AccountUpdateError: String {
         }
     }
 }
+
+enum PasswordChangeError: String {
+    case invalidCsrf = "error.generic.invalid_csrf"
+    case incorectOldPassword = "warning.auth.bad_old_password"
```

Comment:
typo

Reply:
invalidCsrf will be treated as a errorMessage.

Reply:
incorrect instead of incorect

Reply:
resolved with commit [ba1008b](https://github.com/PermanentOrg/permanent-ios/pull/53/commits/ba1008b903c40e28cad6d9989b10ecfd124cd099)